src/modules/Success/SuccessPage.tsx

```diff
@@ -0,0 +1,94 @@
+import styled from "styled-components";
+import { SectionContainer } from "../../components/General.styled";
+import { useContext, useEffect, useState } from "react";
+import { MyContext } from "../../providers/FormValuesContext";
+import { formatKeyAndValue } from "../../Utils/utils";
+import { Span } from "../../components/General.styled";
+
+const SuccessPage = () => {
+  const values = useContext<any>(MyContext);
```

Comment:
great work with the context!
now improve that - export a custom hook on `FormValuesContext.tsx` the return this, instead of using `useContext` directly here.
this also relates to separation of concerns - the SuccessPage should be aware of the framework that provides it with the data (React Context) it should only be aware with `what` it needs. so i would name the hook `useFormDetails` so that its consumer (the SuccessPage) is only familiar with `what` the data it's consuming and not `how` it's implemented

Reply:
Also don't understand. I'll be happy we'll do this together :)